src/app/components/Curation/index.test.tsx

```diff
@@ -4,6 +4,7 @@ import fixture from '../../../../data/pidgin/topics/c95y35941vrt.json';
 import mundoFixture from '../../../../data/mundo/topics/c1en6xwmpkvt.json';
 import kyrgyzHomePage from '../../../../data/kyrgyz/homePage/index.json';
 import { data as kyrgyzMostRead } from '../../../../data/kyrgyz/mostRead/index.json';
+import { radioSchedule as afriqueRadioSchedule } from '../../../../data/afrique/bbc_afrique_radio/scheduleInProcessedShape.json';
```

Comment:
I think it might be better if we used the radio schedule from the afrique homepage fixture data